infra/docker/frontend/prod/Dockerfile

```diff
@@ -17,7 +17,7 @@ COPY libs/@local/design-system/package.json libs/@local/design-system/
 COPY libs/@local/eslint-config/package.json libs/@local/eslint-config/
 COPY libs/@local/hash-isomorphic-utils/package.json libs/@local/hash-isomorphic-utils/
 COPY libs/@local/tsconfig/package.json libs/@local/tsconfig/
-COPY packages/graph/clients/typescript/package.json packages/graph/clients/typescript/
+COPY apps/hash-graph/clients/typescript/package.json apps/hash-graph/clients/typescript/
 COPY apps/hash-frontend/package.json apps/hash-frontend/
```

Comment:
It'd be nice to have these directives alphabetically sorted to ease scanning. We don't have to do this now though.